docs/pom.xml

```diff
@@ -82,11 +77,26 @@ under the License.
 
   <build>
     <plugins>
+      <!-- module with documentation not need to install and deploy -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-install-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Can these skips be done with a property instead? Seems like it would be less config.

Reply:
ok, good idea

Reply:
done